documentation/modules/overview/con-security-configuration-authentication.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// overview/assembly-security-overview.adoc
+
+// UserStory: Securing Kafka -- authentication
+
+[id="security-configuration-authentication_{context}"]
+= Authentication
+Kafka listeners use authentication to ensure a secure client connection to the Kafka cluster.
+
+Supported authentication mechanisms:
+
+* TLS client authentication
+* SASL SCRAM-SHA-512
+* {oauth} token based authentication
+
+The User Operator manages user credentials for authentication.
```

Comment:
Do we need to clarify that this is only SCRAM and TLS, but not OAuth?

Reply:
I think we should clarify, yes. 

The User Operator manages user credentials for TLS and SCRAM authentication, but not {oauth}.